test/cpp/lite_interpreter_runtime/CMakeLists.txt

```diff
@@ -6,13 +6,22 @@ set(LITE_INTERPRETER_RUNTIME_TEST_DIR
   ${TORCH_ROOT}/test/cpp/lite_interpreter_runtime/test_lite_interpreter_runtime.cpp
 )
 
+add_library(backend_with_compiler_runtime SHARED
+        ${TORCH_ROOT}/test/cpp/jit/test_backend_compiler_lib.cpp
+        ${TORCH_ROOT}/torch/csrc/jit/backends/backend_interface.cpp
+        )
+target_compile_definitions(backend_with_compiler_runtime)
+target_link_libraries(backend_with_compiler_runtime torch)
+
 add_executable(
   test_lite_interpreter_runtime
   ${LITE_INTERPRETER_RUNTIME_TEST_DIR})
 target_include_directories(
   test_lite_interpreter_runtime PRIVATE
-  ${ATen_CPU_INCLUDE})
-target_link_libraries(test_lite_interpreter_runtime PRIVATE torch gtest)
+  ${ATen_CPU_INCLUDE}
+)
+
+target_link_libraries(test_lite_interpreter_runtime PRIVATE torch gtest backend_with_compiler_runtime)
```

Comment:
test_lite_interpreter_runtime is linked with torch again, so I'm a bit confused.

Reply:
I could remove torch here but I assume it should be fine, as long as we don't have circular dependency. The reason I keep it here is to make it explicit that test_lite_interpreter_runtime depends on torch. Any potential risk on this way?

Reply:
I see. Probably add a comment so others know why we have it like this?